src/components/results/securityanalysis/security-analysis-table.tsx

```diff
@@ -56,13 +74,24 @@ export const SecurityAnalysisTable: FunctionComponent<
     );
 
     return (
-        <CustomAGGrid
-            rowData={rowsToShow}
-            columnDefs={columnDefs}
-            defaultColDef={defaultColDef}
-            onGridReady={onGridReady}
-            overlayNoRowsTemplate={overlayNoRowsTemplate}
-            {...agGridProps}
-        />
+        <Box sx={styles.gridContainer}>
+            <Box sx={styles.csvExport}>
+                <Box style={{ flexGrow: 1 }}></Box>
+                <ExportButton
+                    disabled={!rowsToShow || rowsToShow.length === 0}
```

Comment:
would it work to do
```suggestion
                    disabled={rowsToShow?.length <= 0}
```
?